docs/getting-started/for-engineers.md

```diff
@@ -6,6 +6,11 @@ slug: /for-engineers
 
 Engineers are typically tasked with implementing an experiment on a new surface with an experiment that has been designed by their product manager in partnership with a data scientist.
 
+## Integrating with a new app
+
+ * [Getting started with Nimbus, for Android Engineers](/getting-started-for-android-engineers)
+ * [Mobile UI needed for Nimbus](http://localhost:3000/getting-started-mobile-required-ui)
```

Comment:
Whenever a user goes to these links, the sidebar disappears. I'm not sure _why_, I think it's because the linked pages aren't in the sidebar tree at all maybe?

If that's expected that's fine, just making sure that was intended.